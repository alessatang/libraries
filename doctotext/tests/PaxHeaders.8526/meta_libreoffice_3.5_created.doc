20 atime=1352037800
29 ctime=1391299512.82451501
