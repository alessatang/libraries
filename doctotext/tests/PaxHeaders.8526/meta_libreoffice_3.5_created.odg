20 atime=1352037800
30 ctime=1391299512.801515407
