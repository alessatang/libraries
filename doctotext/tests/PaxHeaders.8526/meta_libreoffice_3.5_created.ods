20 atime=1352037800
30 ctime=1391299512.775515856
