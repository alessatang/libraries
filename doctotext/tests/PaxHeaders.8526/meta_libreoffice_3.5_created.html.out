30 mtime=1379553524.702593305
30 atime=1389151481.429853122
30 ctime=1389151481.429853122
